pkg/networkmanager/interfaces.js

```diff
@@ -2231,6 +2237,33 @@ PageNetworkInterface.prototype = {
             }
         }
 
+        firewall.addEventListener('changed', function () {
+            if (!firewall.installed) {
+                $('#networking-firewall').hide();
+                return;
+            }
+
+            $('#networking-firewall').show();
+            $('#networking-firewall-switch').onoff('value', firewall.enabled);
+
+            var n = firewall.enabledServices.size;
+            var summary = cockpit.ngettext('$0 Active Rule', '$0 Active Rules', n);
```

Comment:
Sorry, this needs to be wrapped in `cockpit.format(cockpit.ngettext(...), n)` to actually resolve the `$0`.